docs/alert-processing.md

```diff
@@ -0,0 +1,72 @@
+# Alert processing
+
+BOOM consumes Kafka streams of alerts from astronomical surveys
+and outputs Kafka streams for consumers like SkyPortal.
+
+Each alert is processed with the following pipeline:
+
+1. Cross-matches with object IDs from other data catalogs are added.
+   This is done based on the location (right ascension and declination)
+   of the object in the alert. So, for example, if Rubin sends an alert
+   for a given object and then ZTF sends an alert for that same object later,
+   the ZTF alerts will have the object ID from the Rubin catalog.
+   However, the Rubin alert has already been sent out, so it will not have
```

Comment:
I would not mention that, since well we could make that change if we wanted in theory (and we might).